src/main/java/com/mabl/integration/jenkins/MablStepDeploymentRunner.java

```diff
@@ -242,7 +242,12 @@ private void printAllJourneyExecutionStatuses(final ExecutionResult result) {
         for (ExecutionResult.ExecutionSummary summary : result.executions) {
             outputStream.printf("  Plan [%s] is [%s]%n", safePlanName(summary), summary.status);
             for (ExecutionResult.JourneyExecutionResult journeyResult : summary.journeyExecutions) {
-                outputStream.printf("  Journey [%s] is [%s]%n", safeJourneyName(summary, journeyResult.id), journeyResult.status);
+                String journeyFormat = String.format("    Journey [%s] is [%s]", safeJourneyName(summary, journeyResult.id), journeyResult.status);
+                if (journeyResult.status.equalsIgnoreCase("failed")) {
+                    outputStream.printf(journeyFormat + " at [%s]%n", journeyResult.appHref);
```

Comment:
It's fine as is, but to get rid of the concat you could have done `outputStream.printf("%s at [%s]%n", journeyFormat, journeyResult.appHref);`.